types/ibmcloud-appid/index.d.ts

```diff
@@ -0,0 +1,69 @@
+// Type definitions for ibmcloud-appid 6.2
+// Project: https://github.com/ibm-cloud-security/appid-serversdk-nodejs
+// Definitions by: Younes A <https://github.com/younes-io>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface StrategyOptions {
+    [key: string]: any;
+}
+
+export interface SelfServiceOptions {
+    iamApiKey?: string;
+    managementUrl?: string;
+    tenantId?: string;
+    oAuthServerUrl?: string;
+    iamTokenUrl?: string;
+}
+
+export class Strategy {
```

Comment:
you cannot export this as a symmbol, original module does not do this AFAIK:
https://github.com/ibm-cloud-security/appid-serversdk-nodejs/blob/master/lib/appid-sdk.js#L30-L37